src/main.cpp

```diff
@@ -882,6 +891,23 @@ int main(int argc, char** argv) {
       e.saveAudio(outName.c_str(),exportOptions);
       e.waitAudioFile();
     }
+    if (txtOutName!="") {
+      e.setConsoleMode(true);
+      SafeWriter* w=e.saveText(false);
+      if (w!=NULL) {
+        FILE* f=fopen(txtOutName.c_str(),"wb");
+        if (f!=NULL) {
+          fwrite(w->getFinalBuf(),1,w->size(),f);
+          fclose(f);
+        } else {
+          reportError(fmt::sprintf(_("could not open file! (%s)"),txtOutName.c_str()));
```

Comment:
The text in %s is intended to be an error. Change to e.getLastError().

Reply:
I believe that `strerror(errno)` is closer to your intent, see note below.